pgaudit.c

```diff
@@ -1018,15 +1011,18 @@ log_select_dml(Oid auditOid, List *rangeTabls)
         /*
          * We don't have access to the parsetree here, so we have to generate
          * the node type, object type, and command tag by decoding
-         * rte->requiredPerms and rte->relkind.
+         * rte->requiredPerms and rte->relkind. For updates we also check 
+         * rellockmode so that only true UPDATE commands (not
+         * SELECT FOR UPDATE, etc.) are logged.
```

Comment:
I find this comment a bit confusing. As if SELECT FOR UPDATE statements are not logged at all: "only true UPDATE commands are logged".

Reply:
Maybe better with `as UPDATE` at the end?
```
For updates we also check 
* rellockmode so that only true UPDATE commands (not
* SELECT FOR UPDATE, etc.) are logged as UPDATE.
```

Reply:
Yes, much better, thanks.

Reply:
Feel free to approve the PR if everything looks good.